drivers/sx126x/include/sx126x_params.h

```diff
@@ -79,7 +85,9 @@ extern "C" {
                                     .busy_pin = SX126X_PARAM_BUSY,      \
                                     .dio1_pin = SX126X_PARAM_DIO1,      \
                                     .type     = SX126X_PARAM_TYPE,      \
-                                    .regulator = SX126X_PARAM_REGULATOR }
+                                    .regulator = SX126X_PARAM_REGULATOR, \
+                                    .set_rf_mode = SX126X_PARAM_SET_RF_MODE_CB }
```

Comment:
this should be guarded:
```suggestion
#if IS_USED(MODULE_SX126X_RF_SWITCH)
                                    .set_rf_mode = SX126X_PARAM_SET_RF_MODE_CB \
#endif
                                    }
```